plugin/advisor/mysql/mysql_migration_compatibility_advisor_test.go

```diff
@@ -335,3 +335,21 @@ func TestAlterTableChangeColumnType(t *testing.T) {
 
 	runTests(t, tests)
 }
+
+func TestMysql8WindowFunction(t *testing.T) {
```

Comment:
I thought it should be also used in the syntax advisor as well? And the test may be better put there?